app/assets/javascripts/analytics/tracker.js

```diff
@@ -0,0 +1,118 @@
+(function() {
+  "use strict";
+  window.GOVUK = window.GOVUK || {};
+
+  var Tracker = function(universalId, classicId, cookieDomain) {
```

Comment:
Would it be better to make these passed as an object? That way turning off classic analytics would just require us to drop the argument from the code where it's initialised rather than rewrite all of this.